src/main/scala/io/dronekit/cloud/AWSWrapper.scala

```diff
@@ -170,7 +170,7 @@ class AWSWrapper(S3Client: AmazonS3Client = S3.client)
   }
 
   def streamUpload(s3url: S3URL): Sink[ByteString, Future[CompleteMultipartUploadResult]] = {
-    multipartUploadTransform(s3url).to(Sink.head)
+    multipartUploadTransform(s3url).fold(Seq.empty[UploadPartResult])(_ :+ _).to(Sink.head)
```

Comment:
You could choose to keep them from the graph stage if you wanted to use them for something (like creating an MD5, doing additional checks on the upload...). or we could just ignore them. I think either would be ok.